src/ZendeskApi.Client/Resources/UserIdentityResource.cs

```diff
@@ -28,9 +29,7 @@ public IResponse<UserIdentity> Post(UserIdentityRequest request)
 
         public IResponse<UserIdentity> Put(UserIdentityRequest request)
         {
-            var response = Post(request);
```

Comment:
This was like this because you can't update a user's email address (which is something we needed to do) using a put request. The update doesn't work - nothing happens - no errors. 

See https://developer.zendesk.com/rest_api/docs/core/user_identities#allowed-for

It sounds like using a delete, then a post as before is a bad idea because you can't verify a user this way and a user updated using this put method would actually need reverifying. I'll merge this pr.